.github/workflows/pr-lint.yml

```
@@ -25,7 +25,21 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-
+      - name: Run go fmt
+        run: go fmt ./...
+      - name: Run go vet
+        run: go vet ./...
+      - name: Run golangci-lint
+        uses: golangci/golangci-lint-action@v7
+          args: --timeout=5m
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 34: mapping values are not allowed in this context

(syntax-check)